v0.11.0 Release Tour.ipynb

```diff
@@ -0,0 +1,229 @@
+   "id": "hourly-aerospace",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### [#1991](https://github.com/ManimCommunity/manim/pull/1991): Added support for boolean operations on `VMobject`s\n",
+    "\n",
+    "Manim can now compute the union, intersection, and difference of mobjects!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "tutorial-receptor",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING BooleanOperationDemo\n",
+    "\n",
+    "class BooleanOperationDemo(Scene):\n",
+    "    def construct(self):\n",
+    "        c1 = Circle(radius=2, fill_opacity=0.5, color=YELLOW).shift(LEFT)\n",
+    "        c2 = Circle(radius=2, fill_opacity=0.5, color=BLUE).shift(RIGHT)\n",
+    "        circles = VGroup(c1, c2)\n",
+    "        self.add(circles.scale(0.5))\n",
+    "        grid = VGroup(\n",
+    "            Union(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Intersection(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Difference(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Exclusion(c1, c2, color=GREEN, fill_opacity=0.5)\n",
+    "        ).arrange_in_grid(2, 2, buff=3)\n",
+    "        self.add(grid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "shaped-decimal",
+   "metadata": {},
+   "source": [
+    "### [#2118](https://github.com/ManimCommunity/manim/pull/2118): Added 3D support for `ArrowVectorField` and `StreamLines`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "younger-latex",
+   "metadata": {},
+   "source": [
+    "When passing a `z_range` parameter to `ArrowVectorField` or `StreamLines`, the vector field will be rendered in three dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dedicated-market",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING --disable_caching ThreeDArrowVectorField\n",
+    "\n",
+    "class ThreeDArrowVectorField(ThreeDScene):\n",
+    "    def construct(self):\n",
+    "\n",
+    "        def func(p):\n",
+    "            return np.cross(p, np.array([0, 0, p[2]]))\n",
+    "\n",
+    "        vector_field = ArrowVectorField(\n",
+    "            func,\n",
+    "            x_range=[-2.5, 2.5, 1],\n",
+    "            y_range=[-2.5, 2.5, 1],\n",
+    "            z_range=[-1.5, 1.5, 1],\n",
+    "        )\n",
+    "        particle = Dot3D(OUT + RIGHT * 2)\n",
+    "        self.add(vector_field, particle)\n",
+    "        particle.add_updater(vector_field.get_nudge_updater(2))\n",
+    "        self.move_camera(PI/3, -PI/4, run_time=3)\n",
+    "        stream_lines = StreamLines(\n",
+    "            func,\n",
+    "            x_range=[-2.5, 2.5, 1.5],\n",
+    "            y_range=[-2.5, 2.5, 1.5],\n",
+    "            z_range=[-1.5, 1.5, 1.5],\n",
+    "        )\n",
+    "        self.play(Transform(vector_field, stream_lines), run_time=0.5)\n",
```

Comment:
Transform looks a bit weird, maybe one can change it to (FadeOut(vector_field), FadeIn(stream_lines) ?
![image](https://user-images.githubusercontent.com/44469195/135731421-f1b94b51-fa38-45e7-9f5b-09025a2b9efc.png)

Reply:
Changed to `FadeTransform` in af0e905, looks better now. 👍